web/src/Statusbar.scss

```diff
@@ -87,7 +87,6 @@
 // Progress
 .Statusbar-progressPanel {
   width: $sidebar-width - $statusbar-tiltPanel-width;
-  max-width: $sidebar-maxWidth;
```

Comment:
As part of this refactor, I converted Sass to Styled Components and thus reviewed the constants in **constants.scss** to see which ones I could de-dupe into **style-helpers.js**. I can across this one, `$sidebar-maxWidth` which doesn't really seem to be doing much. Both width and max-width are both pixel values, so AFAIK the browser always renders "width" anyway.